spec/features/create_step_by_step_spec.rb

```diff
@@ -18,6 +18,7 @@
     and_I_fill_in_the_form
     and_I_see_a_page_created_success_notice
     and_I_see_I_saved_it_last
+    and_I_can_submit_the_step_by_step_for_2i_review
```

Comment:
Out of scope for this PR maybe, but this seems an odd user journey (being able to submit a step by step for 2i as soon as it's been created). I wonder if there's a more suitable place to put this assertion... for example, after fixing issues identified in the inset-text component?

Reply:
I get your point, but the tests are quite messy and inconsistent at the moment (they're starting to look much better in the past few months though) since we check the actions separately based on document status I wouldn't stress to much on a common scenario at this point, but rather on sanity checks based on states.